packages/p2p/src/components/advertiser-page/advertiser-page.jsx

```diff
@@ -145,7 +149,60 @@ const AdvertiserPage = () => {
                                     </div>
                                 )}
                             </div>
+
+                            {isMobile() && (
+                                <div className='advertiser-page__row'>
+                                    <div className='advertiser-page__rating--row'>
+                                        <OnlineStatusIcon is_online={is_online} />
+                                        <OnlineStatusLabel is_online={is_online} last_online_time={last_online_time} />
+                                    </div>
+                                    <div className='advertiser-page__rating--row'>
+                                        <Text
+                                            className='advertiser-page__joined-since'
+                                            color='less-prominent'
+                                            size={isMobile() ? 'xxxs' : 'xs'}
+                                        >
+                                            {joined_since > 0 ? (
+                                                <Localize
+                                                    i18n_default_text='Joined {{days_since_joined}}d'
+                                                    values={{ days_since_joined: joined_since }}
+                                                />
+                                            ) : (
+                                                <Localize i18n_default_text='Joined today' />
+                                            )}
+                                        </Text>
+                                    </div>
+                                </div>
+                            )}
+
                             <div className='advertiser-page__rating'>
+                                {!isMobile() && (
+                                    <React.Fragment>
+                                        <div className='advertiser-page__rating--row'>
+                                            <OnlineStatusIcon is_online={is_online} />
+                                            <OnlineStatusLabel
+                                                is_online={is_online}
+                                                last_online_time={last_online_time}
+                                            />
+                                        </div>
+                                        <div className='advertiser-page__rating--row'>
+                                            <Text
+                                                className='advertiser-page__joined-since'
+                                                color='less-prominent'
+                                                size={isMobile() ? 'xxxs' : 'xs'}
```

Comment:
same here, we don't need to check isMobile since you're already checking beforehand